javascript/ql/experimental/adaptivethreatmodeling/src/NosqlInjectionATM.md

```diff
@@ -0,0 +1,50 @@
+# NoSQL database query built from user-controlled sources (experimental)
+This is an experimental query. Experimental queries generate alerts using machine learning. They might include more false positives but they will improve over time.
+
+If a database query (such as a SQL or NoSQL query) is built from user-provided data without sufficient sanitization, a malicious user may be able to run malicious database queries.
```

Comment:
Perhaps this should be "If a NoSQL query is built ..."

Reply:
Ah I see the problem ... this is 1 standard query that we split into 2 boosted queries. So we don't have 2 original docs to copy and paste. Therefore please ignore the following comment.